lib/std/zig/llvm/Builder.zig

```diff
@@ -234,8 +234,10 @@ pub const Type = enum(u32) {
     i1,
     i8,
     i16,
+    i24,
```

Comment:
Likewise here, why add these as special cases? Again, `llvm.Builder` already has the capability to represent arbitrary integer bit widths.